tests/libecalc/presentation/exporter/memory_resources.py

```diff
@@ -0,0 +1,157 @@
+from typing import Optional
+
+import pytest
+
+from libecalc.presentation.yaml.yaml_entities import MemoryResource
+
+
+@pytest.fixture
+def generator_electricity2fuel_17MW_resource():
+    return MemoryResource(
+        data=[
+            [0, 0.1, 10, 11, 12, 14, 15, 16, 17, 17.1, 18.5, 20, 20.5, 20.6, 24, 28, 30, 32, 34, 36, 38, 40, 41, 410],
+            [
+                0,
+                75803.4,
+                75803.4,
+                80759.1,
+                85714.8,
+                95744,
+                100728.8,
+                105676.9,
+                110598.4,
+                136263.4,
+                143260,
+                151004.1,
+                153736.5,
+                154084.7,
+                171429.6,
+                191488,
+                201457.5,
+                211353.8,
+                221196.9,
+                231054,
+                241049.3,
+                251374.6,
+                256839.4,
+                2568394,
+            ],
+        ],  # float and int with equal value should count as equal.
+        headers=[
+            "POWER",
+            "FUEL",
+        ],
+    )
```

Comment:
Add a momory_resource_factory in top-level conftest? taking headers and data/columns as input.

It could also be useful to create factories for the specific resources also, so we end up not duplicating the header names (which might change). 

This fixture would then only have to specify power and fuel data into a `electricity2fuel_factory`